StringManip.lua

```diff
@@ -85,24 +85,20 @@ function Serialize.table(input)
     local floor     = math.floor
     local strformat = string.format
     local strfind   = string.find
-    local type      = type
+	local type      = type
```

Comment:
Whitespace got changed for some reason

Reply:
Whitespace was (more generally) messed up in the original PR, likely because I drafted the changes in a separate window then pasted them in. This PR fixes all that as well.